src/pages/gateway/developer-tools.md

```diff
@@ -32,11 +36,19 @@ The [`aio api-mesh:init` command](./command-reference.md#aio-api-meshinit) allow
 
 1. Specify whether you want to use Node Package Manager (`npm`) or `yarn`. (Requires [`npm`](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) or [`yarn`](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable).)
 
-The console indicates that the local environment installed successfully.
+  The console indicates that the local environment installed successfully.
+
+1. To deploy your mesh locally, use the `run` command. You can specify a port by using the `--port` flag or by indicating one in the `.env` file. The port defaults to `5000`.
```

Comment:
How would the port be referenced in the `.env` file?

Reply:
For reference, t he `api-mesh:init` command creates a `.env` file in the project directory. 

Within that file add:
`PORT=9000`